<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataWindow</class>
 <widget class="QMainWindow" name="DataWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelBackg">
    <property name="geometry">
     <rect>
      <x>-210</x>
      <y>-70</y>
      <width>1011</width>
      <height>651</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="images.qrc">:/images/images/wall.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableView" name="qtableA">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="qtableB">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>781</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pButtonGraph">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>530</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboxGraph">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>530</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>WaitingTime</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UsedTime</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>TurnaroundTime</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelTitulo">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Tables</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitulo_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Table A</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTitulo_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data Table B</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>530</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Data Table A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Data Table B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>All Data</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
